llm/server.go

```diff
@@ -166,6 +166,8 @@ func NewLlamaServer(gpus gpu.GpuInfoList, model string, ggml *GGML, adapters, pr
 
 	params = append(params, "--log-disable")
 
+	params = append(params, "--timeout", fmt.Sprintf("%d", 600))
```

Comment:
The server code for the parameter is [here](https://github.com/ollama/ollama/blob/main/llm/ext_server/server.cpp#L2021) and when I run the server with `--help` it reports
```
...
  -to N, --timeout N        server read/write timeout in seconds (default: 600)
...
```

which matches the default value defined [here](https://github.com/ollama/ollama/blob/main/llm/ext_server/server.cpp#L60)

I'm not seeing how this change does anything based on that.  Am I missing something?